operations/notification.go

```diff
@@ -39,10 +40,14 @@ func notificationSlack() cli.Command {
 			},
 		},
 		Action: func(c *cli.Context) error {
-			confPath := c.Parent().String(confFlagName)
+			confPath := c.Parent().Parent().String(confFlagName)
```

Comment:
Was this not working before?

Reply:
It wasn't working before this because `--conf` is specified in the top-level `evergreen` command and not `evergreen notify`